packages/manager/src/features/CloudPulse/Alerts/CreateAlert/utilities.ts

```diff
@@ -38,6 +39,32 @@ export const filterFormValues = (
   };
 };
 
+export const filterEditFormValues = (
+  formValues: CreateAlertDefinitionForm,
+  serviceType: AlertServiceType,
+  severityType: AlertSeverityType,
+  alertId: number
+): EditAlertPayloadWithService => {
+  const values = omitProps(formValues, [
+    'serviceType',
+    'rule_criteria',
+    'trigger_conditions',
+  ]);
+  const entityIds = formValues.entity_ids;
+  const rules = formValues.rule_criteria.rules;
+  const triggerConditions = formValues.trigger_conditions;
+  const severity = formValues.severity ?? severityType;
```

Comment:
```suggestion
  const severity = severityType;
```
We can just pass severityType.

Reply:
@santoshp210-akamai , here we need to rely on formValues.severity to get the edited severity, if it is null somehow, we need to use severity from the alert details

Reply:
Yeah my bad, somehow missed that logic.  Ideally even if it's null, we throw validation error to choose a severity, so we don't accidentally send wrong data as part of the payload. It's just a fallback.

Reply:
Yes..